src/View/Helper/CdnHelper.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace BootstrapUI\View\Helper;
+
+/**
+ * CndHelper class to insert Bootstrap CDN css and javascript assests into layout.
+ *
+ */
+ class CdnHelper extends \Cake\View\Helper
+{
+
+    /**
+     * @var xxx
+     */
+    public $helpers = [
+        'Html'
+    ];
+
+    /**
+     * Default config
+     *
+     * - css: Full URL to most current TwitterBootstrap minified css
+     * - script: Full URL to most current TwitterBootstrap minified js
+     *
+     * @var array
+     */
+    protected $_defaultConfig = [
+        'css' => 'http://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css',
+        'script' => 'http://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js'
```

Comment:
Heh - reading this now, I had the same comment on that one. I was thinking replacing the `3.3.4` by a token `%s` and add a new config key `version` which would replace the token in `getCss()` and `getScript()`.